docs/running-on-yarn.md

```diff
@@ -318,6 +318,14 @@ If you need a reference to the proper location to put log files in the YARN so t
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.executor.failuresValidityInterval</code></td>
```

Comment:
I would keep the original one, since we have an equivalence for AM that is the same name.